pkg/plugin/manager.go

```diff
@@ -89,18 +89,11 @@ func (m *Manager) SetAlias(alias string, correlationID string) error {
 	return nil
 }
 
-func (m *Manager) RemoveAlias(alias string) error {
+func (m *Manager) RemoveAlias(alias string) {
 	m.mutex.Lock()
 	defer m.mutex.Unlock()
 
-	_, exist := m.authorNameToID[alias]
-	if !exist {
-		return fmt.Errorf("while removing alias for %s: no plugin matching the given alias", alias)
-	}
```

Comment:
I'm not sure about that, but the idea is to not block the rest of the process

Reply:
i tried and it seems delete does not error if the key we are deleting does not exist! so you are right this check is useless